setup.py

```diff
@@ -31,7 +31,7 @@
     ]
 
     # use "pip install -e .[dev]" to install required components + extra components
-    data_requires = ["vivarium_cluster_tools>=1.3.6", "vivarium_inputs[data]==4.0.8"]
+    data_requires = ["vivarium_cluster_tools>=1.3.7", "vivarium_inputs[data]==4.0.8"]
     test_requirements = ["pytest", "pytest-mock"]
```

Comment:
You've maybe already considered this, but having these packages on their own lines can make it clear in `git blame` why/when versions get updated.

Reply:
+1